tests/test.sh

```diff
@@ -53,7 +70,7 @@ get_pod_status() {
     # Get the status of the given pod 
     pod=$1
     namespace=${2:-$DEFAULT_NAMESPACE}
-    builtin echo $(kubectl -n ${namespace} get pods $1 -o jsonpath="{.status.phase}")
+    builtin echo $(kubectl get pod $1 -n $namespace -o jsonpath="{.status.phase}")
```

Comment:
Why $namespace better than ?{namespace} ? not sure ...

Reply:
My intention was to eliminate the need to use a specific namespace and replace this argument with the "--all-namespaces" flag. Didn't work so I included this it again. Must have dropped the "{}" inadvertently. Fixed now